converters/provider-aws/common.go

```diff
@@ -0,0 +1,227 @@
+package provider_aws
```

Comment:
Let's remove the underscore:
```suggestion
package provideraws
```

Reply:
Done!